pipelines/_ingredients/cpElectrostaticBFDistortionMatrixLSST.yaml

```diff
@@ -0,0 +1,89 @@
+description: cp_pipe brighter-fatter kernel calibration construction.
+tasks:
+  cpBfIsr:
+    class: lsst.ip.isr.IsrTaskLSST
+    config:
+      connections.ccdExposure: "raw"
+      connections.outputExposure: "cpBfIsrExp"
+      python: |
+        from lsst.cp.pipe import configureIsrTaskLSSTForCalibrations
+
+        configureIsrTaskLSSTForCalibrations(config)
+
+        config.doBootstrap = True
+        config.doCrosstalk = True
+        config.crosstalk.doQuadraticCrosstalkCorrection = False
+        config.doLinearize = True
+        # TODO DM-46426: Add cpCtiLSST pipeline so that this can be True.
+        config.doDeferredCharge = False
+        config.doDefect = True
+        config.doAmpOffset = True
+        config.ampOffset.ampEdgeMaxOffset = 100000.0
+        config.ampOffset.ampEdgeInset = 10
+        config.ampOffset.doBackground = False
+        config.ampOffset.doDetection = False
+        config.ampOffset.doApplyAmpOffset = False
+  cpBfPtcExtractPair:
+    class: lsst.cp.pipe.ptc.PhotonTransferCurveExtractPairTask
+    config:
+      connections.inputExp: "cpBfIsrExp"
+      connections.outputCovariance: "cpBfPtcPartial"
+      maximumRangeCovariancesAstier: 15
+      numEdgeSuspect: 20
+      edgeMaskLevel: "AMP"
+      useEfdPhotodiodeData: true
+      auxiliaryHeaderKeys: ["TEMP6"]
+  cpBfPtcSolve:
+    class: lsst.cp.pipe.ptc.PhotonTransferCurveSolveTask
+    config:
+      connections.inputCovariances: "cpBfPtcPartial"
+      connections.outputPtcDataset: "bfPtcUnadjusted"
+      ptcFitType: "FULLCOVARIANCE"
+      maximumRangeCovariancesAstier: 15
+      maximumRangeCovariancesAstierFullCovFit: 15
+  cpBfPtcAdjustGainRatios:
+    class: lsst.cp.pipe.ptc.PhotonTransferCurveAdjustGainRatiosTask
+    config:
+      connections.exposures: "cpBfIsrExp"
+      connections.input_ptc: "bfPtcUnadjusted"
+      connections.output_ptc: "bfPtc"
+  # TODO DM-46439: This can be renamed back to cpBfSolve when repos
+  # are cleaned up for the previous dimensionality error.
+  cpBfSolveX:
```

Comment:
Has this always been called X?  You can use new labels for the electrostatic mode.  In fact, you should; these are different and not substitutable tasks.